### PR TITLE
Add resolve zendesk tickets content to team manual

### PR DESCRIPTION
**Pull request by jonathanglassman**
Adding content on resolving zendesk tickets to the team manual.

Related trello card: https://trello.com/c/mRUs4FK9/544-add-resolving-zendesk-tickets-content-to-the-team-manual

Related google doc: https://docs.google.com/document/d/1dTX7piv1bgajKl0iuo-NacFOtfyASLxC_7KOOf__kvI/edit?ts=5ff5e2df#heading=h.wtw9x8x5tsx6

### CODE REVIEW

File: source/support/zendesk/index.html.md.erb [hunk 1]
```diff
@@ -1,8 +1,246 @@
...
+We have put in place security measures that defend against the kinds of threats that GOV.UK accounts could face. For example:
+
+- all data is encrypted between us and a user’s web browser using HTTPS
+- a secondary security code is required to log into an account
+- accounts lock out after a set number of failed login attempts
+- we store passwords securely
+
+You can review or report any suspicious activity in the security section of your account.
+
+You also can read about how GOV.UK accounts use cookies and personal information in the full [GOV.UK accounts privacy notice](https://www.gov.uk/government/publications/govuk-accounts-trial-full-privacy-notice-and-accessibility-statement?_ga=2.185125289.1003537497.1608298867-1303934202.1608298867).
```
Comment (huwd) [inline source/support/zendesk/index.html.md.erb]: oops we might want to take the GA param of this link

Comment (huwd) [reply source/support/zendesk/index.html.md.erb]:
```suggestion
You also can read about how GOV.UK accounts use cookies and personal information in the full [GOV.UK accounts privacy notice](https://www.gov.uk/government/publications/govuk-accounts-trial-full-privacy-notice-and-accessibility-statement).
```

**Review by huwd — CHANGES_REQUESTED** [review]
sorry one last bit I didn't spot then happy to approve and merge!

File: source/support/zendesk/index.html.md.erb (hunk 1 above)
Comment (jonathanglassman) [reply source/support/zendesk/index.html.md.erb]: will do

**Review by huwd — APPROVED** [review]
Great work!

**Pull request merged**